frontend/src/scenes/persons/personsLogic.tsx

```diff
@@ -197,7 +197,7 @@ export const personsLogic = kea<personsLogicType>({
                 actions.setPerson({ ...person }) // To update the UI immediately while the request is being processed
                 // :KLUDGE: Person properties are updated asynchronosly in the plugin server - the response won't reflect
                 //      the 'updated' properties yet.
-                await api.persons.update(person.id, person)
+                await api.create(`api/person/${person.id}/update_property`, { key: key, value: newValue })
```

Comment:
I think we moved away from building the api endpoint manually, and instead using the `ApiRequest` builder. Perhaps you can create an explicit `api.persons.updateProperty` method in `api.ts`?

Reply:
the current solution was just copying `delete_property`

Reply:
Cleared up offline: I wasn't suggesting we change the api endpoint, but rather the way we call the endpoint. `api.create` should be replaced with `api.persons.updateProperty` which uses ApiRequest builder to build up the url. There are a few examples in `api.ts::api` you can refer to but let me know if you need any help with this.